TestCases/species_transport/multizone/zone1.cfg

```diff
@@ -0,0 +1,104 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: zone 1, fluid zone                                         %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+SOLVER= INC_RANS
+KIND_TURB_MODEL= SST
+%RESTART_SOL= NO 
+%
+% -------------------- BOUNDARY CONDITION DEFINITION --------------------------%
+%
+MARKER_HEATFLUX= ( wall1, 0.0 )
+MARKER_SYM= ( axis1 )
+SPECIFIED_INLET_PROFILE= NO
+INLET_FILENAME= inlet_venturi.dat
+INC_INLET_TYPE=  VELOCITY_INLET VELOCITY_INLET
+MARKER_INLET= ( gas_inlet1,       300.0, 1.0, 1.0,  0.0, 0.0,\
+                air_axial_inlet1, 305.0, 1.0, 0.0  -1.0, 0.0 )
+SPECIES_USE_STRONG_BC= NO
+MARKER_INLET_SPECIES= (gas_inlet1,       0.5, 0.5,\
+                       air_axial_inlet1, 0.6, 0.0 )	
+MARKER_PLOTTING= (NONE)
+MARKER_MONITORING= (NONE)
+%
+% ---------------- INCOMPRESSIBLE FLOW CONDITION DEFINITION -------------------%
+%
+INC_DENSITY_MODEL= VARIABLE 
+INC_DENSITY_INIT= 1.2
+INC_VELOCITY_INIT= ( 1.0, 0.0, 0.0 )
+INC_ENERGY_EQUATION= YES
+INC_TEMPERATURE_INIT= 300.0
+INC_NONDIM= DIMENSIONAL   
+%
+% -------------------- FLUID PROPERTIES ------------------------------------- %
+%
+FLUID_MODEL= FLUID_MIXTURE   
+MOLECULAR_WEIGHT= (28.96, 25.0, 20.0)
+SPECIFIC_HEAT_CP= (1009.0, 880.0, 800.0)
+CONDUCTIVITY_MODEL= CONSTANT_CONDUCTIVITY
+THERMAL_CONDUCTIVITY_CONSTANT= (0.036, 0.032 0.03)
+%PRANDTL_LAM= 0.72, 0.72, 0.72
+TURBULENT_CONDUCTIVITY_MODEL= NONE
+PRANDTL_TURB= 0.90, 0.90, 0.90
+VISCOSITY_MODEL= CONSTANT_VISCOSITY
+MU_CONSTANT=  (1.716E-5, 1.82e-5, 1.85e-5)
+MIXING_VISCOSITY_MODEL= WILKE
+%
+% ------------- COMMON PARAMETERS DEFINING THE NUMERICAL METHOD ---------------%
+%
+NUM_METHOD_GRAD= GREEN_GAUSS
+CFL_NUMBER= 1000.0
+CFL_ADAPT= NO
+%
+% ------------------------ LINEAR SOLVER DEFINITION ---------------------------%
+%
+LINEAR_SOLVER= FGMRES
+LINEAR_SOLVER_PREC= ILU
+LINEAR_SOLVER_ERROR= 1E-15
+LINEAR_SOLVER_ITER= 5
+%
+% -------------------- FLOW NUMERICAL METHOD DEFINITION -----------------------%
+%
+CONV_NUM_METHOD_FLOW= FDS
+MUSCL_FLOW= YES
+SLOPE_LIMITER_FLOW= NONE
+TIME_DISCRE_FLOW= EULER_IMPLICIT
+%
+% -------------------- SCALAR TRANSPORT ---------------------------------------%
+%
+KIND_SCALAR_MODEL= SPECIES_TRANSPORT
+DIFFUSIVITY_MODEL= CONSTANT_DIFFUSIVITY
+DIFFUSIVITY_CONSTANT= 0.001
+CONV_NUM_METHOD_SPECIES= SCALAR_UPWIND
+MUSCL_SPECIES= NO
+SLOPE_LIMITER_SPECIES = NONE
+SPECIES_INIT= 1.0, 0.0
+SPECIES_CLIPPING= YES
+SPECIES_CLIPPING_MIN= 0.0, 0.0
+SPECIES_CLIPPING_MAX= 1.0, 1.0
```

Comment:
You can have common settings in the master file

Reply:
Good point, I actually get an error from CConfig:SetPostProcessing: "species transport currently only available for compressible and incompressible flow" when I move this to configMaster. It's running late here, so I'll check this error message later, maybe it's just a matter of updating this check...